src/main/java/com/api/entity/SX_Rocket.java

```diff
@@ -1,237 +1,253 @@
 
 @NoArgsConstructor
 @AllArgsConstructor
 @Data
-public class Rocket_DTO {
+public class SX_Rocket {
 
-    @NotEmpty
+    @NotNull
+    @Size(min=5, max=50)
+    public String _id;
+
+    @NotNull
     @Min(1)
-    protected int id;
+    public int id;
 
     @NotNull
-    protected Boolean active;
+    public Boolean active;
 
     @NotNull
     @Min(1)
-    protected int stages;
+    public int stages;
 
     @NotNull
     @Min(1)
-    protected int boosters;
+    public int boosters;
 
     @NotNull
     @Min(1)
     @Max(1000000)
-    protected long cost_per_launch;
+    public long cost_per_launch;
 
     @NotNull
     @Min(1)
-    protected int success_rate_pct;
+    public int success_rate_pct;
 
     @NotNull
     @Pattern(regexp = "\\d{4}-\\d{2}-\\d{2}", message = "Date must be in the format YYYY-MM-DD")
-    protected String first_flight;
+    public String first_flight;
 
     @NotNull
     @Pattern(regexp = "^[a-zA-Z0-9 ]*$", message = "Country must be alphanumeric or spaces")
-    protected String country;
+    public String country;
 
     @NotNull
     @Pattern(regexp = "^[a-zA-Z0-9 ]*$", message = "Company must be alphanumeric or spaces")
-    protected String company;
-    protected DimensionRocket height;
-    protected DimensionRocket diameter;
-    protected MassRocket mass;
-    protected List<PayloadWeights> payload_weights;
-    protected FirstStage first_stage;
-    protected SecondStage second_stage;
-    protected Engines engines;
-    protected LandingLegs landing_legs;
-    protected List<String> flickr_images;
+    public String company;
+    public DimensionRocket height;
+    public DimensionRocket diameter;
+    public MassRocket mass;
+    public List<PayloadWeights> payload_weights;
+    public FirstStage first_stage;
+    public SecondStage second_stage;
+    public Engines engines;
+    public LandingLegs landing_legs;
+    public List<String> flickr_images;
     @NotNull
     @Pattern(regexp = "https://en\\.wikipedia\\.org/", message = "Should be a link from wikipedia")
-    protected String wikipedia;
+    public String wikipedia;
 
     @NotNull
     @Pattern(regexp = "^[a-zA-Z0-9 ]*$", message = "Description must be alphanumeric or spaces")
-    protected String description;
+    public String description;
 
     @NotNull
     @Pattern(regexp = "^[a-zA-Z0-9 ]*$", message = "Rocket id must be alphanumeric or spaces")
     @Size(min = 2, max = 50)
-    protected String rocket_id;
+    public String rocket_id;
 
     @NotNull
     @Pattern(regexp = "^[a-zA-Z0-9 ]*$", message = "Rocket id must be alphanumeric or spaces")
     @Size(min = 2, max = 50)
-    protected String rocket_name;
+    public String rocket_name;
 
     @NotNull
     @Pattern(regexp = "^[a-zA-Z0-9 ]*$", message = "Rocket id must be alphanumeric or spaces")
     @Size(min = 2, max = 50)
-    protected String rocket_type;
+    public String rocket_type;
 
     @Data
+    @AllArgsConstructor
     public static class DimensionRocket {
```

Comment:
move to separate class and add Prexis (even if in the same class child models should have same prefix or suffix)